prestapyt/prestapyt.py

```diff
@@ -319,7 +323,7 @@ def _options_to_querystring(self, options):
         """
         if self.debug:
             options.update({'debug': True})
-        return urllib.urlencode(options)
+        return urllib.parse.urlencode(options)
```

Comment:
Won't work with both py2 and py3: http://python-future.org/compatible_idioms.html#urllib-module